packages/react-icons/package.json

```diff
@@ -25,7 +25,7 @@
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "scripts": {
     "prebuild": "node ./build/generateIcons.js",
-    "build": "yarn build:babel; yarn build:ts",
+    "build": "yarn build:babel && yarn build:ts",
```

Comment:
`&&` has slightly different semantics than `;` here: with this change, `yarn build:ts` will only be executed if `yarn build:babel` succeeds, while it used to be always executed, independently of the return code of `yarn build:babel`.

However, `&&` is supported cross-platform, while the `;` syntax is not.

Considering that stopping the command is probably a sensible thing to do if the babel build fails, I believe that is an acceptable change.

Reply:
nice catch! thanks for your diligent work on this PR @TimoSta and giving us Windows support 👍